presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchConfig.java

```diff
@@ -32,7 +32,8 @@
 {
     public enum Security
     {
-        AWS
+        AWS,
+        PASSWORD,
```

Comment:
should we remove the `,` after `PASSWORD`?

Reply:
Removed.